app/assets/src/components/views/playgrounds/PlaygroundComponents.jsx

```diff
@@ -24,13 +25,49 @@ export default class PlaygroundComponents extends React.Component {
 
   renderComponent = () => {
     const { currentTab } = this.state;
-    if (currentTab === "Geo Search Box") {
+
+    if (currentTab === TABS[0]) {
+      return [
+        // one match
+        <HostOrganismMessage
+          key="1"
+          hostGenomes={[{ name: "Human", id: 1 }]}
```

Comment:
I'm concerned about having the host_genome_ids here. In staging vs. prod I'm pretty sure the host_genome_ids vary already for Human/ERCC/Mosquito. Is there another way?

Reply:
Oh, I see this is just the playground. Doesn't matter then?

Reply:
That's correct, the values in the playground are arbitrary, though I imitated prod values.